frontend/src/components/Teams/MyTeams/partials/CardTeam/BoardsInfo.tsx

```diff
@@ -29,10 +24,10 @@ const BoardsInfo = ({ userSAdmin, teamAdmin }: BoardsInfoProps) => {
 									css={{
 										width: '16px',
```

Comment:
```suggestion
										width: $16,
```
Please verify if this token exists otherwise is good to have it